src/relayclient/RelayProvider.ts

```diff
@@ -130,7 +130,7 @@ export class RelayProvider implements HttpProvider {
           const jsonRpcSendResult = this._convertTransactionToRpcSendResponse(relayingResult.transaction, payload)
           callback(null, jsonRpcSendResult)
         } else {
-          const message = `Failed to relay call. Results:\n${this._dumpRelayingResult(relayingResult)}`
+          const message = `Failed to relay call. Results:\n${_dumpRelayingResult(relayingResult)}`
```

Comment:
this is the only place that ever calls to '_dumpRelayingResult', innit?

Reply:
yes, this and a test.
we could also create it with the formatted string as a member. is it better?